source/renderer/app/components/wallet/tokens/wallet-token/WalletToken.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import '@testing-library/jest-dom';
+
+import React from 'react';
+import noop from 'lodash/noop';
+import { cleanup, screen, waitFor } from '@testing-library/react';
+
+import createTestBed from 'tests/_utils/TestBed';
+import {
+  withDecimalPlacesToken,
+  zeroDecimalPlacesToken,
+} from 'tests/mocks/asset';
+
+import WalletToken from './WalletToken';
+
+const DEFAULT_WALLET_PROPS = {
```

Comment:
A nitpick, but this can be written in `cammelCase`

Reply:
We have consts throughout the app that are in snake case, the only exception is functions.

Reply:
They usually represent a configuration data used globally or at least in few places across the app. In this case this is some data used only within a single file, so I don't se a reason of using the snake case in this case.